ui/build.gradle

```diff
@@ -73,10 +79,10 @@ dependencies {
     annotationProcessor "org.immutables:value:2.7.5"
     testAnnotationProcessor "org.immutables:value:2.7.5"
 
-    implementation('com.google.guava:guava:27.1-android') { force = true }
+    implementation 'com.google.guava:guava:27.1-android'
```

Comment:
Guava is listed in the parent build.gradle ext.versions list, so you should reference that? Also this version definitely needs upgrading, again, happy for that to be a separate PR.

Reply:
Done